docs/data-types/lists.md

```diff
@@ -35,60 +35,60 @@ See the [complete series of list commands](https://redis.io/commands/?group=list
 ## Examples
 
 * Treat a list like a queue (first in, first out):
-```
-> LPUSH work:queue:ids 101
+{{< clients-example list_tutorial queue >}}
+> LPUSH bikes:repairs bike:1
 (integer) 1
-> LPUSH work:queue:ids 237
+> LPUSH bikes:repairs bike:2
 (integer) 2
-> RPOP work:queue:ids
-"101"
-> RPOP work:queue:ids
-"237"
-```
+> RPOP bikes:repairs
+"bike:1"
+> RPOP bikes:repairs
+"bike:2"
+{{< /clients-example >}}
 
 * Treat a list like a stack (first in, last out):
-```
-> LPUSH work:queue:ids 101
+{{< clients-example list_tutorial stack >}}
+> LPUSH bikes:repairs bike:1
 (integer) 1
-> LPUSH work:queue:ids 237
+> LPUSH bikes:repairs bike:2
 (integer) 2
-> LPOP work:queue:ids
-"237"
-> LPOP work:queue:ids
-"101"
-```
+> LPOP bikes:repairs
+"bike:2"
+> LPOP bikes:repairs
+"bike:1"
+{{< /clients-example >}}
 
 * Check the length of a list:
-```
-> LLEN work:queue:ids
+{{< clients-example list_tutorial llen >}}
+> LLEN bikes:repairs
 (integer) 0
-```
+{{< /clients-example >}}
 
 * Atomically pop an element from one list and push to another:
-```
-> LPUSH board:todo:ids 101
+{{< clients-example list_tutorial lmove_lrange >}}
+> LPUSH bikes:repairs bike:1
 (integer) 1
-> LPUSH board:todo:ids 273
+> LPUSH bikes:repairs bike:2
 (integer) 2
-> LMOVE board:todo:ids board:in-progress:ids LEFT LEFT
-"273"
-> LRANGE board:todo:ids 0 -1
-1) "101"
-> LRANGE board:in-progress:ids 0 -1
-1) "273"
-```
-
-* To create a capped list that never grows beyond 100 elements, you can call `LTRIM` after each call to `LPUSH`:
-```
-> LPUSH notifications:user:1 "You've got mail!"
-(integer) 1
-> LTRIM notifications:user:1 0 99
-OK
-> LPUSH notifications:user:1 "Your package will be delivered at 12:01 today."
-(integer) 2
-> LTRIM notifications:user:1 0 99
-OK
-```
+> LMOVE bikes:repairs bikes:finished LEFT LEFT
+"bike:2"
+> LRANGE bikes:repairs 0 -1
+1) "bike:1"
+> LRANGE bikes:finished 0 -1
+1) "bike:2"
+{{< /clients-example >}}
+
+* To limit the length of a list you can call `LTRIM`:
```

Comment:
I don't know why, but this example doesn't display in my local build. You may have to reach out to Igor to solve this.

Reply:
@dwdougherty we resolved this one with the duplicate example yea?

Reply:
@sav-norem Yes.